src/utils/DataProcessingUtils.js

```diff
@@ -113,8 +113,7 @@ function parseEntityAddressKey(entityAddressKey :string) :EntityAddress {
   const split :string[] = entityAddressKey.split(ATAT);
 
   if (split && split.length === 3) {
-    // NOTE: be careful! parseInt() will incorrectly return a number when given certain UUID strings
```

Comment:
someone looking at this in future might wonder what 3 represents. Probably better to declare it as a constant and/or add a comment showing an example of `entityAddressKey` param that the `parseEntityAddressKey` function expects.